src/layout/strategy/container.js

```diff
@@ -1,12 +1,14 @@
 export default function() {
-    var containerWidth = 0,
-        containerHeight = 0;
+    var containerWidth = null,
+        containerHeight = null;
 
     var container = function(point) {
-        return !(point.x < 0 || point.y < 0 ||
+        // If the bounds haven't been defined, then don't enforce
+        return (containerWidth === null && containerHeight === null) ||
```

Comment:
Sorry I know this runs slightly counter to my previous comment but I would always go for ```== null```. I've yet to see a compelling reason to distinguish between the two.